privacy/views/privacy_activity_view.xml

```diff
@@ -43,8 +43,8 @@
                             name="subject_domain"
                             widget="domain"
                             options='{"model": "res.partner"}'
-                            attrs='{"required": [("subject_find", "=", True)],
-                                  "invisible": [("subject_find", "=", False)]}'
+                            required="subject_find == True"
+                            invisible="subject_find == False"
```

Comment:
subject_find is a Boolean, isn't it? in this case we can do `required="subject_find"` and `invisible="not subject_find"`